pkg/package-server/apis/packagemanifest/v1alpha1/types.go

```diff
@@ -0,0 +1,75 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// PackageManifestList is a list of PackageManifest objects.
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type PackageManifestList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+
+	Items []PackageManifest `json:"items"`
+}
+
+// PackageManifest holds information about a package, which is a reference to one (or more)
+// channels under a single package.
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type PackageManifest struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   PackageManifestSpec   `json:"spec,omitempty"`
+	Status PackageManifestStatus `json:"status,omitempty"`
+}
+
+// PackageManifestSpec defines the desired state of PackageManifest
+type PackageManifestSpec struct{}
+
+// PackageManifestStatus represents the current status of the PackageManifest
+type PackageManifestStatus struct {
```

Comment:
We should pull info from the CSV into this. And actually, I think some of it should go into labels if it's not too weird (so that when you do a label query, the object you get back has labels to match)